src/main/resources/db/migration/V73__Add_last_known_addresses_table.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE last_known_address (
+          id UUID PRIMARY KEY,
+          recall_id UUID NOT NULL REFERENCES recall (id),
+          line1 TEXT NOT NULL,
+          line2 TEXT NULL,
+          town TEXT NOT NULL,
+          postcode TEXT NULL,
+          source TEXT NOT NULL,
```

Comment:
Just looking to tighten this to:
          source TEXT NOT NULL CHECK (source IN (('MANUAL','LOOKUP'))),

Reply:
Added a named constraint for this.